.github/images/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+echo -e "\n\n--------- Starting the configuration of rskj ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/rskj.git rskj
+cd rskj && git checkout ${INPUT_RSKJ_BRANCH}
```

Comment:
This is an untrusted input. As such, please quote it.

```suggestion
cd rskj && git checkout "${INPUT_RSKJ_BRANCH}"
```